### PR TITLE
Gv add links

### PR DESCRIPTION
**Pull request by stephenhartman**
Here it is folks, the moment we have all been waiting for...
Links on invalid records!

### CODE REVIEW

File: app/mailers/radbear_mailer.rb [hunk 1]
```diff
@@ -54,7 +55,31 @@ def simple_message(company, recipient, subject, message, options = {})
...
+    @recipient = recipient
+    @problems = problems
+
+    if options[:email_action]
+      @email_action = options[:email_action]
+    end
+
+    if @recipient.respond_to?(:display_name)
+      to_address = "\"#{@recipient.display_name}\" <#{@recipient.email}>"
+    elsif @recipient.class == String
```
Comment (plicjo) [inline app/mailers/radbear_mailer.rb]: Both `User`'s and `Member`'s respond to the `#email` method. Let's reduce this workflow to just emailing to `@recipient.email`

File: app/mailers/radbear_mailer.rb [hunk 2]
```diff
@@ -54,7 +55,31 @@ def simple_message(company, recipient, subject, message, options = {})
...
+      to_address = @recipient
+    else
+      raise "recipient of type #{@recipient.class} if not valid"
+    end
+
+    if options[:from]
```
Comment (plicjo) [inline app/mailers/radbear_mailer.rb]: the `from:` should always be from `no-reply@groundswell.legal` or something like that. Let's nuke this logic.

File: app/mailers/radbear_mailer.rb [hunk 3]
```diff
@@ -54,7 +55,31 @@ def simple_message(company, recipient, subject, message, options = {})
...
     end
   end
-  
+
+  def global_validity(company, recipient, subject, problems, options = {})
+    @company = company
+    @recipient = recipient
+    @problems = problems
+
+    if options[:email_action]
```
Comment (plicjo) [inline app/mailers/radbear_mailer.rb]: What is this doing?

File: app/helpers/radbear_multi_tenant/global_validity_helper.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
+module RadbearMultiTenant
+  module GlobalValidityHelper
+    def global_validity_authorized?(member_or_user, record)
+      member_authorized?(member_or_user, record) || user_authorized?(member_or_user)
+    end
+
+    def global_validity_record_url(record)
```
Comment (plicjo) [inline app/helpers/radbear_multi_tenant/global_validity_helper.rb]: Excellent!

File: app/mailers/radbear_mailer.rb (hunk 3 above)
Comment (stephenhartman) [reply app/mailers/radbear_mailer.rb]: As far as I can tell, the devise mailer uses the email_action option to create a button or specific action related to confirming or unlocking accounts.  We can do without it for the global_validity mailer. 

Example: https://github.com/radicalbear/radbear_multi_tenant/blob/master/app/mailers/radbear_devise_mailer.rb#L17

**Pull request merged**